internal/providers/terraform/google/compute_instance_group_manager.go

```diff
@@ -10,7 +10,10 @@ func getComputeInstanceGroupManagerRegistryItem() *schema.RegistryItem {
 		Name:                "google_compute_instance_group_manager",
 		RFunc:               newComputeInstanceGroupManager,
 		Notes:               []string{"Multiple versions are not supported."},
-		ReferenceAttributes: []string{"version.0.instance_template"},
+		ReferenceAttributes: []string{"version.0.instance_template", "google_compute_per_instance_config.instance_group_manager"},
+		CustomRefIDFunc: func(d *schema.ResourceData) []string {
+			return []string{d.Get("self_link").String(), d.Get("name").String()}
```

Comment:
Nicely done adding the reference for `self_link`.  Am I correct in thinking we should be automatically adding this for every GCE resource (similar to how we automally add `arn` as a reference for AWS resources)?

Reply:
> Nicely done adding the reference for `self_link`. Am I correct in thinking we should be automatically adding this for every GCE resource (similar to how we automally add `arn` as a reference for AWS resources)?

Yes, all google resources have a `self_link` attached (which usually looks like the `id` field), and it could be great to auto add it.

In this specific case, the `name` field is required because that's how the `per_instance_config` is linked to the `group_manager`, but it's kinda exceptional and links usually use `id` or `self_link`.

Reply:
👍  I created a new issue for that.